apps/pbs-docs/src/pages/apis/react.mdx

```diff
@@ -1,38 +1,38 @@
-## `ReactPlainBottomSheet`
+## `BottomSheet`
 
 ### Interface
 
 ```ts
-ReactPlainBottomSheet: (props: PlainBottomSheetProps) => ReactNode;
+BottomSheet: (props: PlainBottomSheetProps) => ReactNode;
```

Comment:
PlainBottomSheeyProps -> BottomSheetProps

Reply:
fixed at 8ba27281581820ea8f0c884a7215e09dcc30c871
Thank you.